system_metrics_collector/src/system_metrics_collector/linux_process_cpu_measurement_node.hpp

```diff
@@ -51,6 +52,9 @@ class LinuxProcessCpuMeasurementNode : public PeriodicMeasurementNode
     const std::string & topic,
     const std::chrono::milliseconds publish_period);
 
+  SYSTEM_METRICS_COLLECTOR_PUBLIC
```

Comment:
I probably actually don't need to do this (and consequently don't need to add the `visibility_control.h` header file to this project) if we're just supporting Linux.

Reply:
This remark should probably be in the comments

Reply:
If the file is already there, we can keep it for now and remove down the line if its causing any problems.

Reply:
I didn't intend to make it sound potentially harmful. I think it's just useless right now since we're only on Linux/ELF where all symbols are just visible by default and we're not doing anything about symbol visibility in our CMake files right now.

Reply:
If not needed then why are we keeping it?

Reply:
Sure. Changes related to symbol visibility have been removed from this pull request.